inventory_management_system_api/routers/v1/item.py

```diff
@@ -54,6 +54,26 @@ def create_item(
         raise HTTPException(status_code=status.HTTP_500_INTERNAL_SERVER_ERROR, detail=message) from exc
 
 
+@router.delete(
+    path="/{item_id}",
+    summary="Delete an item by ID",
+    response_description="Item deleted successfully",
+    status_code=status.HTTP_204_NO_CONTENT,
+)
+def delete_item(
+    item_id: Annotated[str, Path(description="The ID of the item to delete")],
+    item_service: Annotated[ItemService, Depends()],
```

Comment:
```suggestion
    item_service: Annotated[ItemService, Depends(ItemService)],
```

Reply:
There is one like this in `get_item` as well.